pkg/base1/cockpit.js

```diff
@@ -1596,8 +1596,8 @@ function full_scope(cockpit, $, po) {
             } else if (msg.meta) {
                 ensure_cache();
                 $.extend(cache.meta, msg.meta);
-            } else if (msg.owned !== "undefined") {
-                $(self).triggerHandler("owned", [ msg.owned ]);
+            } else if (msg.owner !== "undefined") {
```

Comment:
Why does this work? It seems like this event will be emitted for each message received.